### PR TITLE
optimize the ttl of job

### PR DESCRIPTION
**Pull request by HeGaoYuan**
my implementation refer to 
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/ttlafterfinished/ttlafterfinished_controller.go#L290

### CODE REVIEW

**Review by gaocegege — COMMENTED** [review]
/assign @Jeffwan @johnugeorge

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

File: pkg/controller.v1/common/job.go [hunk 1]
```diff
@@ -45,32 +45,6 @@ func (jc *JobController) DeletePodsAndServices(runPolicy *apiv1.RunPolicy, job i
 	return nil
 }
 
-func (jc *JobController) cleanupJobIfTTL(runPolicy *apiv1.RunPolicy, jobStatus apiv1.JobStatus, job interface{}) error {
```
Comment (johnugeorge) [inline pkg/controller.v1/common/job.go:48]: Which code is referencing this function?

Comment (HeGaoYuan) [reply pkg/controller.v1/common/job.go:48]: @johnugeorge I don't find any code is referencing the cleanupJobIfTTL function. And I found that the cleanupJobIfTTL function is same with the CleanupJob function( before I edit), so I removed it.

Comment (gaocegege) [reply pkg/controller.v1/common/job.go:48]: LGTM, it seems that the func is not used now.

**Review by terrytangyuan — APPROVED** [review]
Thanks!

/lgtm
/approve

**Comment by HeGaoYuan** [comment]
@terrytangyuan @gaocegege 
But I left a todo:  Is the jobStatus.CompletionTime maybe nil ?

**Pull request merged**